.travis.yml

```diff
@@ -23,6 +26,10 @@ env:
     - TEST_COVERAGE=stdout
     - GO_METALINTER_THREADS=1
     - GO_COVER_DIR=_output
+    - VM_DRIVER=none
+    - KUBE_VERSION=v1.13.5
```

Comment:
@Madhu-1 lets put it on `1.14` , or this is the latest from minikube?

Reply:
ddone